[UPDATE] - cypress to v13+

elidakirigo opened the pull request:
`&template=project-enhancement.md`
<!--
  Thank you for submitting a pull request to Checka11y.css!

  We recommend you choose a pull request template by appending the below to the URL above:
  - New A11y Feature: `&template=a11y-feature.md`
  - Bug Fix: `&template=bug-fix.md`
  - Project Enhancement: `&template=project-enhancement.md`

  If none of the pull request templates suit your need, then feel free to this generic template and add as much detail as possible.
-->

## Description

this pull request contains an update of the cypress testing tool to v13+, added files `cypress.config.js` `cypress/e2e/` , updated `packeage.json` and `package.lock.json` deleted `cypress.json`

## why is this important?
this pull request solves the issue #167 

## Covered test cases
<!--
  Were any tests added or modified?
-->

### Did you test on all major browsers?
<!--
  Put an `x` in all the boxes that apply.
  If not, please add a brief explanation as to why you couldn't (e.g. "I couldn't test Safari because I don't have access to an Apple device").
-->
- [x] Chrome
- [x] Firefox
- [x] Edge
- [ ] Safari

## Other details
<!--
  Please add any other details we should be aware of below that don't fit in any of the categories above (e.g. "I would like this pull request to contribute towards my Hacktoberfest contributions").
  If you have nothing to add here, put "N/A".
-->
I would like this pull request to contribute towards my Hacktoberfest contributions
---

## T&Cs
<!--
  Put an `x` in all the boxes that you agree to.
-->

- [x] I confirm I have read and understand the [contributing guidelines](../CONTRIBUTING.md)
- [x] I understand the work in this pull request will not be released straight away and will appear in a future release (if approved)
- [x] I confirm the work in this pull request is true and valid to the best of my knowledge
- [x] I have updated the [README](../README.md), [features.md](../features.md) and [codes.md](../codes.md) files where applicable

jackdomleo7 reviewed (changes requested):
Changes look good, thanks for contributing. However, the tests are appearing to be failing - see [here](https://github.com/jackdomleo7/Checka11y.css/actions/runs/6396670965/job/17541541268?pr=172).

I've also noticed that there are a lot of newly added files which seem to be the same as the existing files. Can we delete some duplicate files?

elidakirigo commented:
thanks, @jackdomleo7 for the feedback. here are the changes I've made 

- on the second commit I was testing out the GitHub CI on my repository (don't mind it)
- on the last commit `[FIX] - all test passing` 
         - I deleted the /cypress/integration folder which was duplicated to /cypress/e2e/ folder.
         - changed the `before` cypress function to `beforeEach` on all specs.
         - updated the `readme.md` file to match the new cypress version v13
         - updated .gitignore to avoid pushing my vscode personal setting, although it would be better to have `.vscode/setting.json` as this would specify the specific settings and not affect other shared vscode settings.

jackdomleo7 reviewed (approved):
Thank you for this contribution! Happy with your recent changes.

Good luck with the rest of Hacktoberfest 🎃
